internal/budgetadjustments/events/update.go

```diff
@@ -62,9 +56,6 @@ func (g *UpdateCmd) run(cmd *cobra.Command) error {
 	if err != nil {
```

Comment:
you can use `yaml.YAMLToJSON()` function here ;) same goes for delete

Reply:
OK